scripts/email/lib.py

```diff
@@ -278,8 +278,8 @@ def _is_allowlisted_sender(self, sender: str) -> bool:
             "erik@bjareho.lt",
             "erik.bjareholt@gmail.com",
             "filip.harald@gmail.com",
-            "bob@superuserlabs.org",
-            "alice@superuserlabs.org",
+            "agent@example.org",
+            "other@example.org",
             "rickard.edic@gmail.com",
         ]
```

Comment:
This should be set from environment variables (about to get upstreamed in improved gptmail version).